src/Microsoft.AspNetCore.Mvc.Formatters.Json/Microsoft.AspNetCore.Mvc.Formatters.Json.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>ASP.NET Core MVC formatters for JSON input and output and for JSON PATCH input using Json.NET.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0618</NoWarn>
```

Comment:
Can you disable warnings at places where the types get used? We shouldn't disable compiler warnings globally if we can avoid it.

Reply:
This is product code. If you haven't updated everything to use the non-`[Obsolete]` constructors, you're not done.

Reply:
I updated majority of tests to use the new constructor except few tests with the old constructors.